src/platforms/common/performance/instrumentation/performance-metrics.mdx

```diff
@@ -31,7 +31,7 @@ description: "Learn how to attach performance metrics to your transactions."
 
 Sentry's SDKs support sending performance metrics data to Sentry. These are numeric values attached to transactions that are aggregated and displayed in Sentry.
 
-<PlatformSection supported={["javascript"]} notSupported={["react-native"]}>
+<PlatformSection supported={["javascript", "android"]} notSupported={["react-native"]}>
```

Comment:
`m`: This should also work on Java, I guess.
```suggestion
<PlatformSection supported={["javascript", "java", "android"]} notSupported={["react-native"]}>
```

Reply:
This section includes automatic performance metrics - in this case I think it's only relevant on Android (slow/cold start, slow/frozen frames).

Custom metrics are mentioned down below and are available for both Java and Android.